x-pack/plugins/security_solution/public/common/components/exceptions/types.ts

```diff
@@ -5,6 +5,8 @@
  */
```

Comment:
NOTE: As mentioned in the 'part 1' PR, these types will get cleaned once the exceptions structure is updated.